packages/gatsby-source-nacelle/README.md

```diff
@@ -0,0 +1,91 @@
+<p align="center">
+  <a href="https://www.getnacelle.com">
+    <img alt="Gatsby" src="https://www.gatsbyjs.com/Gatsby-Monogram.svg" width="60" />
+    <img alt="Nacelle" src="https://raw.githubusercontent.com/getnacelle/gatsby-theme-nacelle/master/examples/store-with-checkout/src/images/nacelle-rocket-icon-crop.png" width="60" />
+  </a>
+</p>
+
+<h1 align="center">
+  Gatsby-Source-Nacelle
+</h1>
+
+<!-- [![npm version](https://img.shields.io/npm/v/@nacelle/gatsby-source-nacelle.svg)](https://www.npmjs.com/package/@nacelle/gatsby-source-nacelle) -->
+<!-- [![GitHub license](https://img.shields.io/github/license/getnacelle/nacelle-react/tree/master/packages/gatsby-source-nacelle)](https://github.com/getnacelle/nacelle-react/tree/master/packages/gatsby-source-nacelle/blob/master/LICENSE) -->
+
+This plugin connects Gatsby to [Nacelle's](https://www.getnacelle.com) Hail Frequency API, which gives you access to the product data (individual products, collections, etc.) and content data (blog posts, articles, etc.) needed to build an eCommerce storefront.
+
+## What is Nacelle?
+
+Nacelle is a headless eCommerce platform made for developers who want to create superior customer buying experiences. When you connect your Shopify, Magento, or custom eCommerce store to Nacelle, our proprietary indexing system supplies a high-performance connection to your back end.
+
+To learn more, check out the [Nacelle docs](https://docs.getnacelle.com/intro.html#what-is-nacelle).
+
+## Quick Start
+
+Follow these steps to add `gatsby-theme-nacelle` to your Gatsby site:
+
+### Install
+
+#### With Yarn
+
+```shell
+yarn add @nacelle/gatsby-theme-nacelle
+```
+
+#### With NPM
+
+```shell
+npm i @nacelle/gatsby-theme-nacelle
+```
+
+### Configure
+
+Then add the theme to your `gatsby-config.js`. Be sure to include your `nacelle-space-id` and `nacelle-graphql-token`, which you can find in your Space settings in the [Nacelle Dashboard](https://dashboard.getnacelle.com/).
+
+#### Adding Your Credentials Directly
```

Comment:
Can we just show the secure option to promote that behavior?

Reply:
?

Reply:
Yeah! Will do.